Scripts/SetAndHandleEmpty/SetAndHandleEmpty.yml

```diff
@@ -0,0 +1,41 @@
+commonfields:
+  id: SetAndHandleEmpty
+  version: -1
+name: SetAndHandleEmpty
+fromversion: 4.5.0
+type: python
+tags:
+- Utility
+comment: Checks if value exists in context. If the value exists it will be set in context,
+  if not nothing will be set.
+enabled: true
+args:
+- name: key
+  required: true
+  default: true
+  description: The key to set
+- name: value
+  description: The value to set to the key. Can be an array. Usually, a dq expression.
+  isArray: true
+- name: append
+  auto: PREDEFINED
+  predefined:
+  - "true"
+  - "false"
+  description: If false then then the context key will be overwritten. If set to true
```

Comment:
Whether to append the new context key to the existing context key. If "false", then the existing context key will be overwritten with the new context key.